### DISCUSSION TITLE
Problem with linking series

### DISCUSSION THREAD

**Discussion by arahmitz** [nunocoracao/blowfish#1274] [Q&A] [answered]
Hey, 
I've noticed one more problem while building for github pages.

I have set my config.toml to add a `series` taxonomy and add `series` with `series_order` properly in the front matter - [can be seen there](https://github.com/arahmitz/adamblogstuff/blob/main/content/posts/squashstretch/index.md) and it works locally, but when I build, seems like the connection breaks - [same post online](https://arahmitz.github.io/adamblogstuff/posts/squashstretch/).

**Comment by ghost** [comment]
u can first set the [seriesOpened](https://github.com/arahmitz/adamblogstuff/blob/main/config/_default/params.toml) in the params.toml

try it

**Reply by arahmitz** [reply]
I believe seriesOpened is used to pre-open series, it doesn't change anything regarding series not being shown (as in not linking) tho.

**Comment by arahmitz** [comment, marked as answer]
I accidentally fixed that by using standard gohugo workflow instead of the docs one. As far as I understand the biggest problem is that action used on gohugo uses "latest" as hugo version and it might impact stuff.
